readme-spir-v-binaries.txt

```diff
@@ -1,17 +1,14 @@
-To run the 2.2 conformance tests test suite for the C++ features you need
+To run the 2.2 conformance tests test suite for the C++ features you need need 
```

Comment:
A single “need” should be enough. :slightly_smiling_face:

Reply:
Thanks for the review. Pushed a small patch to fix that.